src/main/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1116,6 +1116,10 @@ public void setValidationStringency(final ValidationStringency validationStringe
         this.mValidationStringency = validationStringency;
     }
 
+    public boolean hasAttribute(final String tag) {
+        return hasAttribute(SAMTagUtil.getSingleton().makeBinaryTag(tag))!=null;
```

Comment:
shouldn't this just be something like `getAttribute != null?`

Reply:
done